src/controls/qmldir

```diff
@@ -5,6 +5,7 @@ plugin asteroidcontrolsplugin
 CircularSpinner 1.0 qrc:///org/asteroid/controls/qml/CircularSpinner.qml
 PageDot 1.0 qrc:///org/asteroid/controls/qml/PageDot.qml
 PageHeader 1.0 qrc:///org/asteroid/controls/qml/PageHeader.qml
+PercentControl 1.0 qrc:///org/asteroid/controls/qml/PercentControl.qml
```

Comment:
Given that this can get any integer not just percent maybe it'd be better named as IntSelector for example ?

Reply:
Done